### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by plutov**
1. Add errors handling
2. Remove `gorequest` package and use `net/http`
3. Add `options` type
4. Combine `whoisQuery` and `domainQuery` functions into one
5. Add Go 1.10 to Travis.yml :)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+	"fmt"
 	"os"
+
+	"github.com/fatih/color"
 )
 
 func main() {
-
-	if len(os.Args) == 1 {
+	cli := parseArgs(os.Args)
```
Comment (philoserf) [inline main.go]: Perhaps `options` is a better name. `cli` suggests something more and less than these options.

Comment (philoserf) [reply main.go]: I can find examples of options structs in the wild that look much like this struct if you like, or need persuasion.

Comment (plutov) [reply main.go]: Thanks. Do you mean only var name or type name also?

Comment (philoserf) [reply main.go]: Both. I meant to comment the struct too.

Not a material difference, i know. I think it improves the story the new code tells.

**Comment by philoserf** [comment]
@TimothyYe LGTM

**Comment by TimothyYe** [comment]
Nice job! Thank you for your contribution!

**Pull request merged**